zebrad/src/commands/start.rs

```diff
@@ -275,6 +293,81 @@ impl StartCmd {
             ),
         )
     }
+
+    /// Logs Zebra's estimated progress towards the chain tip.
+    async fn update_progress(
+        network: Network,
+        latest_chain_tip: impl ChainTip,
+        sync_status: SyncStatus,
+    ) {
+        // The amount of time between progress logs.
+        const LOG_INTERVAL: Duration = Duration::from_secs(60);
+
+        // The number of blocks we consider to be close to the tip.
+        //
+        // Most chain forks are 1-7 blocks long.
+        //
+        // TODO: reduce this number after slow syncing near tip gets fixed (#3375)
+        const MAX_CLOSE_TO_TIP_BLOCKS: i32 = 20;
```

Comment:
I just saw the "stalled sync" warning at 30 blocks, so I'm roughly doubling that amount:

```suggestion
        const MAX_CLOSE_TO_TIP_BLOCKS: i32 = 60;
```